src/Cli/dotnet/Commands/New/BuiltInTemplatePackageProvider.cs

```diff
@@ -19,14 +17,16 @@ internal sealed class BuiltInTemplatePackageProvider(BuiltInTemplatePackageProvi
 
     public ITemplatePackageProviderFactory Factory { get; } = factory;
 
-#pragma warning disable CS0067
     /// <summary>
     /// We don't trigger this event, we could complicate our life with FileSystemWatcher.
-    /// But since "dotnet new" is short lived process is not worth it, plus it would cause
-    /// some perf hit...
+    /// But since "dotnet new" is short lived process is not worth it, plus it would cause some perf hit...
+    /// To avoid warnings about being unused, implement empty add/remove accessors.
     /// </summary>
-    public event Action TemplatePackagesChanged;
-#pragma warning restore CS0067
+    public event Action? TemplatePackagesChanged
+    {
+        add { }
+        remove { }
+    }
```

Comment:
This solution was used in another file to avoid that CS0067 warning.  Seems reasonable to me.